lvgl_shell.html

```diff
@@ -1,4 +1,18 @@
 <html>
+	<head>
+		<style type="text/css">
+			html, body {
+			  margin: 0;
+			}
+
+			body {
+			  height: 100%;
```

Comment:
In my experience, you need full width and height on both `body` and `html` for it to actually center on all browsers.

```css
html, body {
    width: 100%;
    height: 100%;
    min-width: 100%;
    min-height: 100%;
}
```

Reply:
Done